requirements.txt

```diff
@@ -1,5 +1,6 @@
-django>=2.2,<3.2
-django-leaflet>=0.23,<0.28
-channels>=2.3.0,<2.5.0
-Pillow~=7.2.0
+django>=2.2,<3.1
+django-leaflet>=0.23,<0.27
+channels>=2.1.3,<2.5.0
+Pillow>=6.2.0,<=7.1.0
```

Comment:
Could you rebase on master so to include this change?
https://github.com/openwisp/django-loci/commit/a8cfe17a8c7d6291dc02f2192fe979cfbd0b13d7